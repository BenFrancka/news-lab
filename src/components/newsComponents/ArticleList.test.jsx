import dotenv from 'dotenv';
dotenv.config();
import React from 'react';
import { render, screen } from '@testing-library/react';
//import ArticleContainer from '../../containers/ArticleContainer';
import ArticleList from './ArticleList';

describe('Article List', () => {
  it('should be comprised of an ul', async () => {
    const articles = [
      {
        title: 'title1',
        author: 'author1',
        description: 'description1',
      },
      {
        title: 'title2',
        author: 'author2',
        description: 'description2',
      },
    ];
    render(<ArticleList articles={articles} />);

    const ul = await screen.findByRole('list', { name: 'articles' });

    expect(ul).toMatchSnapshot();
  });
});
