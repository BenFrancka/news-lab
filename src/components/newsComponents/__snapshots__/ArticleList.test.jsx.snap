// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article List should be comprised of an ul 1`] = `
<ul
  aria-label="articles"
>
  <li>
    <h2>
      title1
    </h2>
    <h3>
      author1
    </h3>
    <p>
      description1
    </p>
  </li>
  <li>
    <h2>
      title2
    </h2>
    <h3>
      author2
    </h3>
    <p>
      description2
    </p>
  </li>
</ul>
`;
